src/enums/priority-code.ts

```diff
@@ -0,0 +1,5 @@
+/** Enum for Priority codes. */
+export enum PriorityCode {
+  N = 'N',
+  Y = 'Y'
+}
```

Comment:
Out of curiosity, why the enum for yes/no vs a boolean property?

Reply:
The API expects to see "Y" or "N" here, and I didn't like the UI code to assign or compare strings, so I created this enum to enforce type-checking on the subject variable.

Reply:
Sounds good, no prob!

Interesting that the api would expect an 'N' or 'Y' however 😝

Reply:
🤷‍♂️